### PR TITLE
Web NFC: Convert Chromium Layout tests to testharness tests

### PR DESCRIPTION
**Pull request by Honry**
- Convert most of Layout tests from https://chromium.googlesource.com/chromium/src/+/master/third_party/WebKit/LayoutTests/nfc/
 - Add some manual tests to check nfc push/watch with different types of records

### CODE REVIEW

**Comment by alexshalamov** [comment]
@Honry Sorry for delay, I was OOO. Did you try running those tests on platform that has webnfc? E.g., Chromium on Android?

**Comment by Honry** [comment]
Yes, all tests work fine on latest Chrome Canary(62.0.3165.0) for Android.

File: web-nfc/nfc_block_iframe.https.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+<div id="log"></div>
+
+<script>
+
+'use strict';
+
+async_test( t => {
+  let iframe = document.createElement('iframe');
+  iframe.src = 'resources/push-from-iframe.html';
```
Comment (alexshalamov) [inline web-nfc/nfc_block_iframe.https.html]: nit: This can be inlined, like for sensors iframe test.

**Review by alexshalamov — APPROVED** [review]
lgtm with nit

File: web-nfc/nfc_block_iframe.https.html (hunk 1 above)
Comment (Honry) [inline web-nfc/nfc_block_iframe.https.html]: @alexshalamov, thanks for the reminder, done the optimization.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258401831)

# Firefox (nightly) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258401832)

# Sauce (safari) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258401840)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/258401841)

# Sauce (MicrosoftEdge) #

**Comment by Honry** [comment]
@alexshalamov, I've fixed the nit, is it OK for you now?

**Review by alexshalamov — APPROVED** [review]

**Comment by Honry** [comment]
@alexshalamov, thanks for your review, I will merge it directly.

**Pull request merged**